### PR TITLE
FEAT: Swagger 2.x to OpenAPI 3.

### PR DESCRIPTION
**Pull request by carlwilson**
Used swagger tools to convert and replace `api/par-api.yaml`.

Requires #56 to be merged first.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,107 +1,120 @@
...
+
 - [`pip`](https://pypi.org/project/pip/) to install Python dependencies;
 - [`demjson`](https://pypi.org/project/demjson/) for command line JSON linting;
 - [`jsonschema`](https://pypi.org/project/jsonschema/) for JSON schema validation;
 - [`pytest`](https://pypi.org/project/pytest/) to run the schema validation tests.
 
 Once you have Pythton and pip you can install the other dependencies so: `pip install -r requirements.txt`
 
+If the installation of demjson fails with an error similar to the below:
+
```
Comment (jackdos) [inline README.md:64]: assuming you were intending to document an error case?

**Review by jhsimpson — APPROVED** [review]
This looks good to me. I'd like to figure out how to show this in the swagger-ui in github pages, but that could be a separate PR.

**Review by jackdos — COMMENTED** [review]
Hard to really look in detail at the main change, but loading it in the swagger editor tool it looks believable. Just that dangling comment in the README

File: README.md (hunk 1 above)
Comment (carlwilson) [reply README.md:64]: Hah, good catch but should have left that out.

**Pull request merged**